terraform/bastion/Jenkinsfile

```diff
@@ -75,14 +84,17 @@ pipeline {
         //no-color option set for Terraform commands as Jenkins console unable to output the colour
         //making output difficult to read
         TF_CLI_ARGS="-no-color"
+        TF_VAR_connect_to_database="${params.CONNECT_TO_DATABASE}"
+        TF_VAR_connect_to_backend_checks_efs="${params.CONNECT_TO_BACKEND_CHECKS_EFS}"
+        TF_VAR_connect_to_export_efs="${params.CONNECT_TO_EXPORT_EFS}"
       }
       stages {
         stage('Set up Terraform workspace') {
           steps {
             script {
               dir("terraform/bastion") {
                 echo 'Initializing Terraform...'
-                sh "git clone https://github.com/nationalarchives/tdr-terraform-modules.git"
+                sh "git clone --branch connect-to-efs-from-bastion https://github.com/nationalarchives/tdr-terraform-modules.git"
```

Comment:
Should the `--branch` param be in the command?

Reply:
Nope, that was just for testing. I've taken it out.